README.md

```diff
@@ -1,25 +1,47 @@
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 Configures a domain hosted on Route53 to work with AWS Simple Email Service (SES)
 
-Creates the following resources:
+## Prerequisites
+* Ensure [terraform](https://www.terraform.io/intro/getting-started/install.html) is installed
+* Ensure domain is registered in [route53](https://aws.amazon.com/route53/)
+* Ensure an s3 bucket exists and SES has write permissions to it
+* If you have an existing rule set you can skip creating the dependent resource
+* route53 zone id can be obtained by looking up the domain in route53 service
+
+## Getting Started
+1. Import the module called `ses_domain` and update its source property to `github.com/trussworks/terraform-aws-ses-domain` and run `terrafrom init`
+2. The next step is to configure the module with [minimum values](#usage) for SES to start working
+3. Once fully configured run `terraform plan` to see the execution plan and `terrafrom apply` to stand up SES
+
+**Creates the following resources:**
 
 * MX record pointing to AWS's SMTP endpoint
 * TXT record for SPF validation
 * Custom MAIL FROM domain
 * CNAME records for DKIM verification
 * SES Verfication for the domain
 
-NOTE: SES is only available in us-east-1, us-west-2, and eu-west-1
+### NOTES: 
+* SES is only available in us-east-1, us-west-2, and eu-west-1
+* SES out of the box locks the service in development mode; please see this documentation on how to make it production ready. Until the service is in production mode you can only send emails to confirmed email accounts denoted in `from_addresses`
 
 ## Usage
 
 ```hcl
 module "ses_domain" {
   source = "../../modules/aws-ses-domain"
+  domain_name        = "example.com"
+  mail_from_domain   = "email.example.com"
+  route53_zone_id    = "Z123456789"
```

Comment:
I think I'd include the data source lookup above and reference it here. Up to you.

Reply:
> For the zone id?

yep makes sense that's what we ended up doing in our infra code